website/docs/r/vpn_gateway_nat_rule.html.markdown

```diff
@@ -58,16 +64,40 @@ The following arguments are supported:
 
 * `vpn_gateway_id` - (Required) The ID of the VPN Gateway that this VPN Gateway NAT Rule belongs to. Changing this forces a new resource to be created.
 
-* `external_address_space_mappings` - (Required) A list of CIDR Ranges which are used for external mapping of the VPN Gateway NAT Rule.
+* `external_address_space_mappings` - (Optional) A list of CIDR Ranges which are used for external mapping of the VPN Gateway NAT Rule.
```

Comment:
```suggestion
* `external_address_space_mappings` - (Deprecated) A list of CIDR Ranges which are used for external mapping of the VPN Gateway NAT Rule.
```

Reply:
Let's also move `external_address_space_mappings` and `internal_address_space_mappings` to the bottom to get them out of view.

Reply:
Updated